pandas/tests/indexes/datetimes/test_indexing.py

```diff
@@ -600,3 +600,24 @@ def test_get_loc_nat(self):
         # GH#20464
         index = DatetimeIndex(['1/3/2000', 'NaT'])
         assert index.get_loc(pd.NaT) == 1
+
+    def test_getitem_bool_index(self):
+        # GH#22533
+        idx1 = pd.date_range('2011-01-01', '2011-01-05', freq='D', name='idx')
```

Comment:
Rather than listing these out sequentially can you parametrize the various cases? Would make the code more succinct and readable